### ISSUE TITLE
About the test accuracy on CUB.

### ISSUE THREAD

**Issue by Hoodythree** [Tsingularity/PoseNorm_Fewshot#1]
Great work!But I can only get 45.791% accuracy when tested on CUB using proto+pn, while that is 49.56% in the paper. Is there any hyperparameter tuning needed here to reach the accuracy in the paper?

**Comment by Tsingularity** [comment]
Hi thanks a lot for reading our paper and using our codes!

Are you using the default hyper-parameters in Conv4.sh? If yes, I would say I have never got such accuracy on my end. Could you please share more details about your environment including pytorch and cudnn version?

On our end, under the experiment environment described in Readme.md, we ran 8 times with different random seeds and the lowest accuracy is 47.98 while the highest is 51.01, others are basically around 49.50. Before we release the code, we also re-run the experiments under new up-to-date pytorch/cudnn versions (pytorch 1.3.0 and cudnn 7603) and we did find that there's about 1~1.5 points performance decrease but it's across all the methods including the baselines. For proto+pn, we re-run 3*8=24 times with different random seeds and the average is 48.4. The lowest accuracy is 46.12 while the highest is 49.96 and others are basically round the average acc. We currently don't know why the version difference could influence the few-shot performance but it seems doesn't affect the performance gap between baselines and PN models.

Also given 1) CUB is a fairly small dataset (compared to mini-ImageNet) 2) the samples for meta-test are very few, i think several points of variance is acceptable as long as it is across all the methods. This is also found in the CUB experiment in "closer look at few-shot classification" paper and its github discussions (in their [readme](https://github.com/wyharveychen/CloserLookFewShot#faq), FAQ 1). In order to reduce such variance, I would recommend to re-run several times with different random seeds and see how it goes.

Hope this would help.
